packages/api/models/mappings.json

```diff
@@ -29,7 +29,13 @@
         "type": "text"
       },
       "executions": {
-        "type": "text"
+        "type": "text",
```

Comment:
this field would ideally be `keyword`

Reply:
I don't think you can even do that with reindexing. Once you have an indexed field in elastic, you can't change it. You can add a new mapping (i.e. executions_keyword) and index that as well and use that for the search. 

I'm not opposed to reindexing necessarily, but it's nice to be able to save the end user from having to do that.